pkg/server/http.go

```diff
@@ -104,6 +105,7 @@ func newHTTPProxy(ctx context.Context, config *HTTPConfig, grpcServer *grpcServe
 	handler = promhttp.InstrumentHandlerDuration(metrics.httpLatency, handler)
 	handler = promhttp.InstrumentHandlerCounter(metrics.httpRequestsCount, handler)
 	handler = promhttp.InstrumentHandlerRequestSize(metrics.httpRequestSize, handler)
+	handler = clog.WithCloudTraceContext(handler)
```

Comment:
To confirm, is cloud tracing something that needs to be handled in gRPC as well, or instead in gRPC?

Reply:
its a good catch, let me see how the outbound load balancers handle that for gRPC traffic